maas/client/viscera/vlans.py

```diff
@@ -49,12 +49,12 @@ class Vlan(Object, metaclass=VlanType):
     """A VLAN in a fabric."""
 
     id = ObjectField.Checked(
-        "id", check(int), readonly=True)
+        "id", check(int), readonly=True, pk=0)
```

Comment:
This is wrong as well. A VLAN is unique based on this `fabric_id` and its `id`. That is why its pk=0 and pk=1.